lib/IRGen/CMakeLists.txt

```diff
@@ -57,6 +58,8 @@ add_swift_host_library(swiftIRGen STATIC
   Outlining.cpp
   StructLayout.cpp
   SwiftTargetInfo.cpp
+  TBDGen.cpp
+  TBDGenRequests.cpp
```

Comment:
IMO it would be nice the TBDGen files could be put into a TBDGen folder within IRGen, just so there's at least some logical separation between them and the rest of IRGen